tasks/install.yml

```diff
@@ -5,6 +5,7 @@
       - postgresql
       - postgresql-client
       - postgresql-doc
+      - libpq-dev
```

Comment:
Why is this needed?

Reply:
I got an error when running synapse_port_db without this.